File: go/raindrops/raindrops.go
package raindrops

import "strconv"

// Convert outputs a string which is different depending on num's factors
func Convert(num int) string {
	var output string
	if num%3 == 0 {
		output += "Pling"
	}

	if num%5 == 0 {
		output += "Plang"
	}

	if num%7 == 0 {
		output += "Plong"
	}

	if len(output) == 0 {
		output += strconv.Itoa(num)
	}

	return output
}

File: python/raindrops/raindrops.py
def convert(number):
    '''
    Take a number and return a string depending on its factors.
    '''
    result = ""
    if number % 3 == 0:
        result += "Pling"

    if number % 5 == 0:
        result += "Plang"

    if number % 7 == 0:
        result += "Plong"

    if len(result) == 0:
        result += f"{number}"

    return result
